packages/core/src/lib/cli/internal/get-entry-from-cli-or-config.ts

```diff
@@ -26,3 +41,34 @@ export function getEntryFromCliOrConfig(entryFile = '', runInit = true): Project
 
   throw new Error('Please provide an entry file, e.g. main.ts');
 }
+
+
+// Helper function to process entry file consistently
+function processEntryFile(
+  entryFileValue: string | Record<string, string>,
+  runInit: boolean,
+  fs: ReturnType<typeof getFs>
+): Array<ProjectEntry<ProjectInfo>> | Array<ProjectEntry<string>> {
+  if (typeof entryFileValue === 'string') {
+    return runInit
+      ? [
+          {
+            projectName: DEFAULT_PROJECT_NAME,
```

Comment:
hint for reviewer
For single project workspaces I am assigning here a constant for the project name so that I can later in e.g. verify easily know if where are inside a single or multi project workspace